docker/Dockerfiles/Dockerfile-dev

```diff
@@ -1,54 +1,33 @@
-################################################################################
-# Build stage 0 `builder`:
-# Install Rally from source inside a virtualenv
-################################################################################
+FROM docker.elastic.co/wolfi/python:3.11.7-dev
+ARG RALLY_VERSION
+ARG RALLY_LICENSE
 
-FROM python:3.8.13-slim-bullseye as builder
+ENV RALLY_RUNNING_IN_DOCKER=True
 
-RUN apt-get -y update && \
-    apt-get install -y curl git gcc && \
-    apt-get -y upgrade && \
-    rm -rf /var/lib/apt/lists/*
+USER root
 
-RUN mkdir -p /rally/esrally
-COPY pyproject.toml /rally/
-COPY README.md /rally/
-COPY esrally/ /rally/esrally/
+RUN apk update 
+RUN apk add curl git gcc pigz bash
```

Comment:
Could we add zstd too? Rally supports that and some of ours tracks use zstd compression for artifacts

Reply:
That seems completely reasonable